src/main/frontend/src/components/AddCommentForm.js

```diff
@@ -4,10 +4,100 @@
 //
 
 class AddCommentForm extends React.Component {
-	
+	constructor(props) {
+		super(props);
+		this.state = {
+			comment: '',
+			validationErrors: [],
+			isDisabled: true
+		};
+		this.handleSubmit = this.handleSubmit.bind(this);
+		this.handleChange = this.handleChange.bind(this);
+	}
+	handleChange(event) {
+		event.preventDefault();
+		this.setState({
+			isDisabled: false,
+			comment: event.target.value
+		});
+	}
+
+	handleSubmit(event) {
+		event.preventDefault();
+
+		axios
+			.patch(this.props.url, [
+				{
+					op: 'add',
+					path: '/comment',
+					value: this.state.comment
+				}
+			])
+			.then(response => {
+				const data = response.data;
+
+				if (data.hasOwnProperty('fieldErrors')) {
+					const fieldErrors = [];
+					if (data.fieldErrors.comment) {
+						fieldErrors.push(...data.fieldErrors.comment);
+					}
+					this.setState({
+						isDisabled: false,
+						validationErrors: fieldErrors
+					});
+					return;
+				}
+
+				// no need to reset the state as page will be reloaded
+				window.location.reload();
+			})
+			.catch(error => {
+				console.error('err', error);
+				this.setState({ isDisabled: false, hasServerError: true });
+			});
+	}
 	render() {
+		const hasValidationErrors = this.state.validationErrors.length > 0;
 		return (
-			<div></div>
-		)
+			<div className="col-sm-12 form-group">
+				<form className="form-horizontal" onSubmit={this.handleSubmit}>
```

Comment:
In order to make an errored field to be highlighted properly we also should have `has-error` class on a form:
```suggestion
				<form className={`form-horizontal ${hasValidationErrors ? 'has-error' : ''}`} onSubmit={this.handleSubmit}>
```